data/incipias/first contact.txt

```diff
@@ -0,0 +1,300 @@
+			choice
+				"Because you do not want to separate species by force?"
+					goto separation
+				"Is it not a nice thought of species peacefully making contact?"
+					goto peace
+				"It is always better when species interact with each other early peacefully than making contact late but forcefully."
+					goto early
+			label separation
+			`	You explain the Hai that you belief in a peaceful united galaxy where all species interact with each other. After you told that to the Hai he says that he has to talk to some people. "You have to wait here for a while." He says and leaves the room.`
+			`	This time you have to wait longer. But when he comes back he has good news for you. "We do not think that you lied about your intentions, but before we help you you have to give us a proof of at least one other peaceful contact you made to a species. If you can bring us this proof we will see how we can help you.`
+				goto offer
+			label peace
+			`	You explain that you like to make contact with new species and the cultural exchange. You tell the Hai of the positive impacts on both sides of a peaceful contact. The Hai listens carefully and after you have finished he says "Now I understand your noble reasons for this. I will have to go to talk to some people." He leaves the room and tells you to wait there.`
+			`	You have to wait a longer period of time until he comes back. "I have good news for you. We believe you that your intentions are only this peaceful ones. We will help you if you fulfill the following condition: Bring us one other proof of a peaceful contact you made to another species."`
+				goto offer
+			label early
+			`	You explain to the Hai that you think it is best to make contact to a species as soon as possible. Because at some point in the distant future there will be contact with them anyways so it is now the best time to establish peaceful connections. After the Hai hears that he stands up and says "I will talk to some people about the things you told me. Please wait here." and leaves the room.`
+			`	A long time passes before the Hai comes back. When he comes back he sits down and says "We are mostly sure now that you have no bad intentions. However, before we start helping you with this matter we want something from you that proves you made peaceful contact to another species. If you can get this for us we will see how we can help you.`
+			label offer
+			`	You ask the Hai what this proof could look like. The Hai answers "That is something I have no answer for. Maybe it would be best if you talk to the government of the species. It was a pleasure to talk to you. I will see you when you have the proof. Oh, and one last thing: It might be better to not tell this species exactly why you need this." With these words he guides you to the exit and you say goodbye to him.`
```

Comment:
This whole request for proof from another species is also kinda trite.
Realistically the Hai don't actually care that much about other species.

When dealing with the Hai you should think of them as an iron fist in a velvet glove.

They're absolutely more interested in friendly relations than they are in fighting. Cooperation is the big ticket for modern Hai society... but their society is built on an absolutely ironclad bureaucracy that has been grown, refined, fortified, and entrenched over such a length of time that we only invented the word 'bureaucracy' somewhere in the last 1% of it.

The public reasoning that the Hai give for any particular act or decision might be any manner of positively spun propaganda - but make no mistake, the bureaucracy endorses no person to action that the Hai do not benefit from in some way.